client/src/pages/ProjectDetail.js

```diff
@@ -3,61 +3,90 @@ import { Link } from "react-router-dom";
 import { Col, Row, Container } from "../components/Grid";
 import Jumbotron from "../components/Jumbotron";
 import API from "../services/API";
+// import TestTile from "../components/TestTile";
+import Tiles from "../components/Tiles";
+// import Project from '../components/Project';
+import Moment from 'react-moment';
 
 class ProjectDetail extends Component {
   state = {
     project: {},
-  
+
+
   };
 
   // When this component mounts, grab the PROJECT with the id of this.props.match.params.id
- //(this.props.match.params.id) <--- is how we get the ID from URL
+  //(this.props.match.params.id) <--- is how we get the ID from URL
   componentDidMount() {
-    console.log(this.props.match.params.projectId);
+    // console.log(this.props.match.params.projectId);
     API.getProjectDetails(this.props.match.params.projectId)
-      .then(res => this.setState({project: res.data}))
+      .then(res =>
+        this.setState({
+          project: res.data,
+          benefactorName: res.data.Benefactor.name,
+          benefactorDescription: res.data.Benefactor.description,
+          category: res.data.Benefactor.name
+        })
+      )
       .catch(err => console.log(err));
   }
 
   render() {
-    console.log(this.state.project);
+    // console.log(this.state.project);
     return (
       <Container fluid>
         <Row>
           <Col size="md-12">
             <Jumbotron>
-             
+
               <h2>
-                Learn More About {this.state.project.title}
+                Learn More About Lending Your Time to {this.state.benefactorName}
```

Comment:
comment this out?

Reply:
this should stay, it pulls benefactor info from project table via benefactor table for line 26 on ProjectDetail.js